services/sellerService.js

```diff
@@ -166,16 +164,16 @@ module.exports = {
         });
     },
 
-    updateService: function(params) {
+    updateService: function({ email, serviceId, ...rest }) {
         return new Promise((resolve, reject) => {
-            if (!params.email || !params.serviceId) {
+            if (!email || !serviceId) {
                 reject('Missing Params');
             } else {
-                validationService.doesSuchServiceExist(params)
+                doesSuchServiceExist(params)
                     .then(result => {
                         if (result) {
                             const service = result;
-                            service.updateAttributes(params)
+                            service.updateAttributes(rest)
```

Comment:
updating rest params because serviceId and email are primary key (can't update). nice :)